src/yang/ietf-twamp.yang

```diff
@@ -642,66 +660,68 @@ module ietf-twamp {
-                
-              The configured (or auto-allocated) value is advertized
-              in the Sender Port field of the Request-TW-session
-              message (see Section 3.5 of RFC 5357). Note that
-              in the scenario where a device auto-allocates a UDP
-              port number for a session, and the repeat parameter
-              for that session indicates that it should be
-              repeated, the device is free to auto-allocate a
-              different UDP port number when it negotiates the
-              next (repeated) iteration of this session.";
+               the Session-Sender for this TWAMP-Test session.
+               The number is restricted to the dynamic port range.
+
+               By default the Control-Client SHALL auto-allocate a
+               UDP port number for this TWAMP-Test session.
+
+               The configured (or auto-allocated) value is advertized
+               in the Sender Port field of the Request-TW-session
+               message (see Section 3.5 of RFC 5357). Note that
+               in the scenario where a device auto-allocates a UDP
+               port number for a session, and the repeat parameter
+               for that session indicates that it should be
+               repeated, the device is free to auto-allocate a
+               different UDP port number when it negotiates the
+               next (repeated) iteration of this session.";
           }
 
           leaf reflector-ip {
             type inet:ip-address;
             mandatory true;
             description
               "The IP address belonging to the remote
-              Session-Reflector device to which the TWAMP-Test
-              session will be initiated. This value will be
-              used to populate the receiver address field of
-              the Request-TW-Session message.";
+               Session-Reflector device to which the TWAMP-Test
+               session will be initiated. This value will be
+               used to populate the receiver address field of
+               the Request-TW-Session message.";
           }
 
           leaf reflector-udp-port {
-            type dynamic-port-number;
+            type uint32 {
+              range "862 | 49152..65535";
+            }
             description
               "This parameter defines the UDP port number that
-              will be used by the Session-Reflector for
-              this TWAMP-Test session. The number is restricted
-              to the dynamic port range and is to be placed in
-              the Receiver Port field of the Request-TW-Session
-              message.";
+               will be used by the Session-Reflector for
+               this TWAMP-Test session. The default number is within
+               to the dynamic port range and is to be placed in
+               the Receiver Port field of the Request-TW-Session
+               message.";
```

Comment:
I'd like to add "The new well-known port number (862) MAY be used." to the description, but a new PR might create a merge conflict (?)

Reply:
Ok. Will add.